ietf/doc/views_help.py

```diff
@@ -1,5 +1,7 @@
 # Copyright The IETF Trust 2013-2023, All Rights Reserved
 
+import debug # pyflakes: ignore
```

Comment:
Did you forget to remove this?

Reply:
No - it's convention to leave it once we add it to a module to assist with future development. That's why it has the the `ignore` pyflakes directive.